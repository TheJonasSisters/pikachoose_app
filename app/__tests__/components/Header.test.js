import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Header from '../../javascript/components/components/Header'

Enzyme.configure({ adapter: new Adapter() });

it('Header component has Navbar component', () => {
  const wrap = shallow(<Header/>)
  expect(wrap.text()).toEqual('<Navbar />')
})

describe(`Component: Header`, () => {
  test(`Header renders with default props`, () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
