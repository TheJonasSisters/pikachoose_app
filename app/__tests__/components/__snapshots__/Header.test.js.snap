// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Header Header renders with default props 1`] = `ShallowWrapper {}`;
